overrides/about.mdx

```diff
@@ -16,9 +16,9 @@ description: "Visualization, Exploration, and Data Analysis (VEDA): Scalable and
 
     As an interactive visual interface for storytelling, the dashboard provides a number of capabilities including sharing and interacting with datasets and their respective stories. Through visualizations, trend analyses, and comparative analyses of datasets, users can explore the applications and implications of those data. VEDA Dashboard will improve the usability of the Earth science data and enable faster science – one of the goals of the NASA open-source science initiative. Please submit any feedback or questions using the Feedback button in the top right of the dashboard.
 
-    ## Commitment to Open Source
+    ## Commitment to Open Science
 
-    The NASA open-source science initiative believes commitment to open source is an important aspect of creating and maintaining high-quality software. Open source allows for collaboration and transparency, which can lead to better solutions and more robust code.
+    In alignment with NASA's [Transform to Open Science](https://science.nasa.gov/open-science/transform-to-open-science) (TOPS) initiative, VEDA believes in a commitment to open source, open data, and open science is an important aspect of creating and maintaining high-quality software. Open source allows for collaboration and transparency, which can lead to better solutions and more robust code.
```

Comment:
With the change, this sentence "...VEDA believes in a commitment to open source, open data, and open science is an important aspect of creating and maintaining high-quality software." has a strange ring to it.

Perhaps "believes that a commitment"